### PR TITLE
Rane Wallin - MVP

### PR DESCRIPTION
**Pull request by RaneWallin**

### CODE REVIEW

**Comment by RaneWallin** [comment]
MVP Complete

File: index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const server = require('./server');
```
Comment (GannonDetroit) [inline index.js]: good job

File: posts/post-routes.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+const express = require("express");
+
+const db = require("../data/db");
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
```
Comment (GannonDetroit) [inline posts/post-routes.js]: excellent job. I know this is super similar to the previous assignment but backend is all about getting in reps and reps and reps on these endpoints. So good job on doing that and adding in the router concept.

File: server.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const PostRouter = require('./posts/post-routes');
+const server = express();
+
```
Comment (GannonDetroit) [inline server.js]: good job

**Pull request merged**